cg/services/order_validation_service/workflows/mutant/constants.py

```diff
@@ -4,7 +4,7 @@
 
 
 class MutantDeliveryType(Enum):
-    ANALYSIS_FILES = DataDelivery.FASTQ_ANALYSIS
+    ANALYSIS_FILES = DataDelivery.FASTQ
```

Comment:
it will be at least from when I merge the PRs related to that. Likely going to happen this week

Reply:
Tracking in this issue that this is in place before any final merges: https://github.com/Clinical-Genomics/improve-order-flow/issues/99. I do not think it is a main concern for this PR but it will of course need changing after those merges. Can be beneficial for tests to have it like this until then, though. Especially given that there is no option in the order portal for other options yet